### PR TITLE
[TIMOB-25396] Update Ti.Geolocation API-URL

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25396

### CODE REVIEW

**Comment by ewieberappc** [comment]
FR Passed. Ran multiple geolocation/geocoder test/suites. All data came back as expected.

**Review by ewieberappc — APPROVED** [review]
FR Passed

File: iphone/Classes/GeolocationModule.m [hunk 1]
```diff
@@ -47,7 +47,7 @@ - (void)dealloc
 - (void)start:(NSDictionary *)params
 {
   // https://api.appcelerator.net/p/v1/geo
```
Comment (sgtcoolguy) [inline iphone/Classes/GeolocationModule.m]: https://api.appcelerator.com/p/v1/geo

Comment (hansemannn) [reply iphone/Classes/GeolocationModule.m]: Too late to edit.

**Pull request merged**